examples/tutorial/jupyter/execution/omnisci_on_native/local/exercise_3.ipynb

```diff
@@ -0,0 +1,139 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "![LOGO](../../../img/MODIN_ver2_hrz.png)\n",
+    "\n",
+    "<center><h2>Scale your pandas workflows by changing one line of code</h2>\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Exercise 3: Not Implemented\n",
+    "\n",
+    "**GOAL**: Learn what happens when a function is not yet supported in Modin as well as how to extend Modin's functionality using the DataFrame Algebra."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "When functionality has not yet been implemented, we default to pandas\n",
+    "\n",
+    "![](../../../img/omnisci_convert_to_pandas.png)\n",
+    "\n",
+    "We convert a Modin dataframe to pandas to do the operation, then convert it back once it is finished. These operations will have a high overhead due to the communication involved and will take longer than pandas.\n",
```

Comment:
This should be elaborated with respect to OmniSci engine.

Reply:
@YarShev This was expansion